crates/tools/js/benchmark/src/index.ts

```diff
@@ -391,10 +393,6 @@ function preprocessConfig(config: any) {
     config.providerConfig.hardfork
   );
 
-  // "accounts" in EDR are "genesisAccounts" in Hardhat
```

Comment:
We no longer have `genesisAccounts` in EDR. Instead we have a `genesisState` but Hardhat doesn't expose this in its config.

Reply:
I assume this was left over from a previous change?

Reply:
Correct. The benchmark code wasn't updated in the past, after that field was removed.